pkg/meta/sql.go

```diff
@@ -750,66 +577,22 @@ func (m *dbMeta) Lookup(ctx Context, parent Ino, name string, inode *Ino, attr *
 		return errno(err)
 	}
 	if !exist {
-		if m.conf.CaseInsensi {
-			// TODO: in SQL
-			if e := m.resolveCase(ctx, parent, name); e != nil {
-				*inode = e.Inode
-				return m.GetAttr(ctx, *inode, attr)
-			}
-		}
 		return syscall.ENOENT
 	}
 	*inode = nn.Inode
 	m.parseAttr(&nn.node, attr)
 	return 0
 }
 
-func (r *dbMeta) Resolve(ctx Context, parent Ino, path string, inode *Ino, attr *Attr) syscall.Errno {
-	return syscall.ENOTSUP
-}
-
-func (m *dbMeta) Access(ctx Context, inode Ino, mmask uint8, attr *Attr) syscall.Errno {
-	if ctx.Uid() == 0 {
-		return 0
-	}
-
-	if attr == nil || !attr.Full {
-		if attr == nil {
-			attr = &Attr{}
-		}
-		err := m.GetAttr(ctx, inode, attr)
-		if err != 0 {
-			return err
-		}
-	}
-
-	mode := accessMode(attr, ctx.Uid(), ctx.Gid())
-	if mode&mmask != mmask {
-		logger.Debugf("Access inode %d %o, mode %o, request mode %o", inode, attr.Mode, mode, mmask)
-		return syscall.EACCES
-	}
-	return 0
-}
-
-func (m *dbMeta) GetAttr(ctx Context, inode Ino, attr *Attr) syscall.Errno {
-	inode = m.checkRoot(inode)
-	if m.conf.OpenCache > 0 && m.of.Check(inode, attr) {
-		return 0
-	}
-	defer timeit(time.Now())
+func (m *dbMeta) getAttr(ctx Context, inode Ino, attr *Attr) syscall.Errno {
 	var n = node{Inode: inode}
 	ok, err := m.engine.Get(&n)
-	if err == nil && ok {
-		m.parseAttr(&n, attr)
-		m.of.Update(inode, attr)
-	} else if inode == 1 {
-		err = nil
-		attr.Typ = TypeDirectory
-		attr.Mode = 0777
-		attr.Nlink = 2
-		attr.Length = 4 << 10
-	} else if err == nil {
-		err = syscall.ENOENT
+	if err == nil {
```

Comment:
I think it will be better like: 
```
	if err != nil {
		return errno(err)
	}
	if ok {
		m.parseAttr(&n, attr)
		return syscall.F_OK
	}
	return syscall.ENOENT
```